docker-gen.go

```diff
@@ -72,6 +72,7 @@ type RuntimeContainer struct {
 	Volumes      map[string]Volume
 	Node         SwarmNode
 	Labels       map[string]string
+	Server       ServerInfo
```

Comment:
I think that's a great idea. Can you get to the Context from inside a template, though? It looks like it only sends an array of containers to the generateFile function.